templates/react-ecs/mprocs.yaml

```
@@ -8,3 +8,16 @@ procs:
   anvil:
     cwd: packages/contracts
     shell: anvil --base-fee 0 --block-time 2
+  indexer:
+    shell: rimraf $SQLITE_FILENAME && pnpm sqlite-indexer
+    env:
+      RPC_HTTP_URL: "http://127.0.0.1:8545"
+      FOLLOW_BLOCK_TAG: "latest"
+      SQLITE_FILENAME: "indexer.db"
+  explorer:
+    cwd: /Users/karolis/Code/Lattice.xyz/experiments/mud-2/packages/explorer
+    shell: pnpm dev
```

actually maybe instead of changing `cwd`,  what we want is this to be
```suggestion
    shell: cd ../../packages/explorer && pnpm dev
```

then we can use a relative path to indexer DB

---

I'll try this and consult with you when updating templates, should have not added this now, my bad